app.js

```diff
@@ -0,0 +1,217 @@
+// Global Array to store player objects
```

Comment:
**Kindly remove commented code, you can always use git history to access previous code right :slightly_smiling_face:**